frontend/src/components/Header.tsx

```diff
@@ -3,14 +3,13 @@ import {useContext} from "react";
 import AuthContext from "../context/authentication/AuthContext";
 
 export default function Header() {
-    const {userLoggedIn, logout} = useContext(AuthContext);
-
+    const {loggedInUser, logout} = useContext(AuthContext);
 
     return (
 
         <header>
-            <MyNavBar loggedInUser={userLoggedIn} logout={logout}/>
+            <MyNavBar loggedInUser={loggedInUser} logout={logout}/>
```

Comment:
If only the navbar uses these why not use the context inside the navbar? Saves one layer of indirection